geometry_msgs/PoseStamped[] latest_poses
Velocity[] latest_velocities